### PR TITLE
RL_UserCard

### PR DESCRIPTION
**Pull request by lluneva**

### CODE REVIEW

**Comment by lluneva** [comment]
With static data

**Review by janisknets — APPROVED** [review]

File: src/App.js [hunk 1]
```diff
@@ -2,11 +2,12 @@ import React, { Component } from "react";
 import "./App.css";
 import Header from "./components/Header/Header";
 
+
 class App extends Component {
   render() {
     return (
       <React.Fragment>
```
Comment (rokija) [inline src/App.js]: why is React.Fragment used here?

File: src/App.js [hunk 2]
```diff
@@ -2,11 +2,12 @@ import React, { Component } from "react";
 import "./App.css";
 import Header from "./components/Header/Header";
 
+
```
Comment (rokija) [inline src/App.js:5]: remove empty line

File: src/components/Usercard/Usercard.css [hunk 3]
```diff
@@ -0,0 +1,21 @@
+.UserCard {
+  align-items: left;
+  text-align: left;
```
Comment (rokija) [inline src/components/Usercard/Usercard.css]: why both align-items and text-align?

File: src/components/Usercard/Usercard.css [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  padding-top: 1%;
+  border-bottom: 1px solid rgb(230, 227, 227);
+}
```
Comment (rokija) [inline src/components/Usercard/Usercard.css]: add new line

File: src/components/Usercard/Usercard.css [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+.Usercard__head {
+  display: flex;
+}
```
Comment (rokija) [inline src/components/Usercard/Usercard.css]: add new line

File: src/components/Usercard/Usercard.css [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+.UserCard__username {
+  margin-right: 20px;
+  margin-bottom: 0;
```
Comment (rokija) [inline src/components/Usercard/Usercard.css]: remove this

File: src/components/Usercard/Usercard.css [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+  font-size: 1rem;
+}
```
Comment (rokija) [inline src/components/Usercard/Usercard.css]: new line

File: src/components/Usercard/Usercard.css [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+.UserCard__content {
+  font-size: 12px;
+}
```
Comment (rokija) [inline src/components/Usercard/Usercard.css]: new line

File: src/components/Usercard/Usercard.css [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+.UserCard__editBtn {
```
Comment (rokija) [inline src/components/Usercard/Usercard.css]: rename to UserCard__edit-button

File: src/components/Usercard/Usercard.js [hunk 10]
```diff
@@ -0,0 +1,33 @@
+import React, { Component } from "react";
+import "./Usercard.css";
+
+class Usercard extends Component {
+  render() {
+    const { username, email, level, id, goToUserId } = this.props;
+
+    return (
+      <React.Fragment>
```
Comment (rokija) [inline src/components/Usercard/Usercard.js]: why is React.Fragment used here?

File: src/components/Usercard/Usercard.js [hunk 11]
```diff
@@ -0,0 +1,33 @@
...
+        <div className="UserCard">
+          <div className="">
```
Comment (rokija) [inline src/components/Usercard/Usercard.js]: remove this div if its not needed

File: src/components/Usercard/Usercard.js [hunk 12]
```diff
@@ -0,0 +1,33 @@
...
+            <div className="Usercard__head">
+              <h4 className="UserCard__username">{username}</h4>
+              <button
+                className="UserCard__editBtn btn btn-secondary"
+                onClick={goToUserId}
```
Comment (rokija) [inline src/components/Usercard/Usercard.js]: what is this function doing? is it not editUser?

File: src/components/Userlist/Userlist.js [hunk 13]
```diff
@@ -0,0 +1,67 @@
+import React, { Component } from "react";
+import Usercard from "../Usercard/Usercard";
+
+class Userlist extends Component {
```
Comment (rokija) [inline src/components/Userlist/Userlist.js]: rename to UserList

File: src/components/Usercard/Usercard.js (hunk 12 above)
Comment (rokija) [reply src/components/Usercard/Usercard.js]: ah I got it, it goes to a separate page where the user is displayed?

File: src/components/Userlist/Userlist.js [hunk 14]
```diff
@@ -0,0 +1,67 @@
...
+  goToUserId = () => {
+    this.props.history.push("/:userid");
+  };
+
+  render() {
+    const { users } = this.state;
+    return (
+      <div className="Userlist">
+        <h2> Users</h2>
+        {/* {this.props.users && this.props.users.map((user, index) */}
```
Comment (rokija) [inline src/components/Userlist/Userlist.js]: remove commented out code

**Review by rokija — CHANGES_REQUESTED** [review]
please add human readable title and description to this pull request

**Comment by rokija** [comment]
screenshot 2019-02-19 at 17 16 31

**Comment by rokija** [comment]
screenshot 2019-02-20 at 09 33 58

if I click on the edit button

**Review by rokija — APPROVED** [review]

**Pull request merged**